/* Copyright 2023 The FedLearner Authors. All Rights Reserved.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *    http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

syntax = "proto3";
package pp_lite.proto;

enum DataJoinType {
  HASHED_DATA_JOIN = 0;
  OT_PSI = 1;
}

enum FileType {
  CSV = 0;
  TFRECORD = 1;
}

message Ping {
  string message = 1;
}

message Pong {
  string message = 1;
}
